portfolio/src/main/java/com/google/sps/data/Refrigerator.java

```diff
@@ -24,35 +24,81 @@
  */
 public class Refrigerator {
     //refrigerator fields
-    private int id;
-    private String owner;
+    private String id;
+    private User owner;
     private ArrayList<Item> items;
+    private ArrayList<User> users;
 
     //class constructor
-    public Refrigerator(int id, String owner, ArrayList<Item> items) {
+    public Refrigerator(String id, User owner, ArrayList<Item> items, ArrayList<User> users) {
         this.id = id;
         this.owner = owner;
         this.items = items;
+        this.users = users;
+        users.add(owner);
     }
 
     //getter and setter methods
-    public int getID(){
+    public String getID(){
         return id;
     }
 
-    public String getOwner(){
+    public User getOwner(){
         return owner;
     }
 
-    public void setID(int idNumber){
+    public ArrayList<Items> getItems() {
+        return items;
+    }
+
+    public ArrayList<User> getUsers(){
+        return users;
+    }
+
+    public void setID(String idNumber){
         id = idNumber;
     }
 
-    public void setOwner(String newOwner){
+    public void setOwner(User newOwner){
         owner = newOwner;
     }
 
     public void addNewItem(Item newItem){
         items.add(newItem);
     }
+
+    public void addUser(User newUser){
+        users.add(newUser)
+    }
+
+    //method for getting the expiry date of a single item in the fridge
```

Comment:
Nice use of comments to describe what a function is doing for the reader. Javadoc conventions for method comments are generally like this:


/** This is a description of what this method does. */